docs/guides/genomics-platform/requesting-data/about-our-data.md

```diff
@@ -12,30 +12,79 @@ St. Jude Cloud hosts both raw genomic data files and processed results files:
 
 ### gVCF files
 
 We provide gVCF files produced by the [Microsoft Genomics Service][msgen]. gVCF files are derived from the BAM files produced above as called by [GATK's haplotype caller][gatk-haplotype-caller]. Today, we defer to [the official specification document][gvcf-spec] from the Broad Institute, as well as [this discussion][gvcf-diff-from-vcf] on the difference between VCF and gVCF files. For more information about how Microsoft Genomics produces gVCF files or any other questions regarding data generation, please refer to [the official Microsoft Genomics whitepaper][msgen-whitepaper].
 
 ### Somatic VCF files
 
-Somatic VCF files contain HG38 based SNV/Indel variant calls from the St. Jude somatic variant analysis pipeline as follows. Broadly speaking:
+Somatic VCF files available on St. Jude Cloud have been generated when possible for tumor samples where a matching normal sample has also been sequenced. These include samples from the Pediatric Cancer Genome Project (PCGP), Clinical Pilot, and Genomes 4 Kids (G4K) [cohorts][pediatric-cancer-cohorts]. These somatic VCF files contain HG38 coordinate SNV/Indel variant calls generated by the St. Jude somatic variant analysis pipeline outlined below:
 
-1. Reads were aligned to HG19 using [bwa backtrack][bwa] (`bwa aln` + `bwa sampe`) using default parameters.
+1. Whole genome (WGS) and/or Whole exome (WES) sequencing data were aligned to HG19 using [bwa backtrack][bwa] (`bwa aln` + `bwa sampe`) using default parameters.
 2. Post processing of aligned reads was performed using [Picard][picard] `CleanSam` and `MarkDuplicates`.
-3. Variants were called using the [Bambino][bambino-paper] variant caller (you can download Bambino [here][bambino-download] or by navigating to the [Zhang Lab page][bambino-program] where the  "Bambino package" is listed as a dependency under the CONSERTING section).
-4. Variants were post-processed using an in-house post-processing pipeline that cleans and annotates variants. This pipeline is not currently publicly available.
-5. Variants were manually reviewed by analysts and published with [the relevant Pediatric Cancer Genome Project (PCGP) paper][pcgp-landing-page].
-6. Post-publication, variants were lifted over to HG38 (the original HG19 coordinates are stored in the `HG19` INFO field.).
+3. Variants were called using the [Bambino][bambino-paper] variant caller (you can download Bambino [here][bambino-download]).
+4. If RNA-Sequencing was performed on a tumor sample, we assess the presence of any called variants in a StrongARM aligned RNA-Seq BAM file.
+5. In some Clinical Pilot cases, validation by custom capture was performed, in which case the read counts of that experiment are reported.
+6. Variants were post-processed using an in-house post-processing pipeline that cleans and annotates variants. This pipeline is not currently publicly available.
```

Comment:
We should cite the ClinicalPilot paper as it describes the in-house pipeline: https://www.nature.com/articles/s41467-018-06485-7

Reply:
We do, but do you want it more prominent? After listing the steps, it says `For more detailed information on steps 1-6, see the [Clinical Pilot paper][clinical-pilot] where they are described in depth.`

Reply:
After I made the comment, I did notice that we referenced the paper below. I do think making it more prominent would be helpful. Maybe even marking each step with an asterisk or something to indicate a footnote that says detailed description in this paper.

Reply:
Also, worth noting that the G4K manuscript is currently in the review process. We'll potentially want to include a reference to this paper when It comes out.